Project_2/README.md

```diff
@@ -1,16 +1,76 @@
 # GroupAwesome-AwesomeProject1
 
-Some of the problem i had while trying to make this for java:
-That the question makes me print things from the consumer and the job, which means I can't even use interfaces.
+## 1. Accomplished Items  
 
+**Java program**
+1. Monitor implemented
+2.  
 
-some problems i had with c, it not opp which is a pain. the pointer are a real pain as its hard to know what to use and where is is var, *var, **var, &var. 
-memory on the stack was a huge pain. I had object overrideing each other due to a thread being slow and that part of the stack was "not longer in use" so a new object just when there. 
-C has shit error messages, like its the same as a http 500 error, basicly something when wrong. To top that it throws error randomly sometime.
- Like no joke i was in the main method with no threads on a for loop and it was like ahhhh this increment causes a segfault, and i was like dufuk C. 
-The documentation is complete garbage, like its hard to find how to use anything or how it interacts with anything, basicly its just a list of function and you have to brute force 
-your way to find the function that your want. 
+**C Program**
+1. Producer production with wait time
+2. Consumer consumption with wait time
+3. Process sleeps to allow for process completions
+4. Formatted output file as per requirements
+5. Implemented with pthreads library
 
+## 1.1 Build instructions   
 
+**C Program**
+1. Clone repo
+2. Navigate inside `GroupAwesome-AwesomeProject1` directory
+3. Compile program by running `gcc main.c`
 
-configeration for each part can be found at the top of the main file.
\ No newline at end of file
+  - CMakeList already contains information that allows compilation including queue.c and pthread handling4
+  - If you are on different compiler/platforms and run into error due to pthread. Try the following commands to:
+
+| Compiler/Platform | Compilar command |
+| ----------------- | ---------------- |
+| GNU Linux         | gcc -pthread     |
+| Intel Linux       | icc -pthread     |
+| PGI Linux         | pgcc -lpthread   |
+
+4. Run the compiled program using the following command `./main`
+
+**Java program**  
+1. Clone repo
+2. Navigate inside `GroupAwesome-AwesomeProject1` directory
+3. Compile program by running
+
+
+## 1.2 Contributions
```

Comment:
I think this should go at the start of the readme, It would make is easier for the TA to know who the group members are.